dockerfiles/Dockerfile.react-native-cli

```diff
@@ -0,0 +1,27 @@
+# CI test image for building the CLI
+FROM node:lts-alpine as react-native-cli-feature-builder
+
+WORKDIR /app
+
+COPY package*.json ./
+RUN npm install
+
+COPY babel.config.js lerna.json .eslintignore .eslintrc.js jest.config.js tsconfig.json packages/expo/unimodules-core.d.ts ./
+COPY bin ./bin
+COPY packages ./packages
+
+RUN npm run bootstrap -- --concurrency 1
+
+RUN npx lerna run build --scope bugsnag-react-native-cli
+RUN npm pack --verbose packages/react-native-cli/
+
+# The maze-runner test image
+FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:integration-interactive_cli-cli as react-native-cli-maze-runner
+
+WORKDIR /app
+
+COPY test/react-native-cli test/react-native-cli
+COPY --from=react-native-cli-feature-builder /app/bugsnag-react-native-cli-*.tgz ./
+RUN for d in test/react-native-cli/features/fixtures/*/; do cp /app/*.tgz "$d"; done
```

Comment:
We'll want this to install from the package we publish to Artifactory, although that may be a task for when it's integrated into the pipeline.  Perhaps you could add a TODO here for now?

Reply:
...the benefit of having it like this for now is that it will just work when running locally.  To run from a published artifact we'd need to facilitate and document overriding the version (like we've done for React Native).

Reply:
This is copied from [the node dockerfile](https://github.com/bugsnag/bugsnag-js/blob/next/dockerfiles/Dockerfile.node#L28-L29); is that changing too?

Reply:
Eventually, yes, there's a ticket to update it at a later point.

Reply:
The history on this was that we could use `pack` for Node and it was easier - but React Native forced our arm into using the published artifacts, due to Lerna not providing a `pack` command.  Publish/install is more representative of real life and of course we'd like it to be eventually consistent.